### ISSUE TITLE
9.0.1 npm package does not support require/import in Markdown examples

### ISSUE THREAD

**Issue by rgrwkmn** [styleguidist/react-styleguidist#1298]
**Current behavior**

When installing v9.0.0 or v9.0.1 from npm, I am not able to use require or import to include other JS files in Markdown examples. If I git clone the react-styleguidist repo directly (master branch) and npm link it to my project, then it works.

**To reproduce**

https://github.com/rgrwkmn/styleguidist-require-import-bug

**Expected behavior**

I want to import other JS files in my Markdown examples according to the documentation.

**Comment by stepankuzmin** [comment]
Same issue here. But I could import external libraries, but couldn't import local components.

**Comment by stepankuzmin** [comment]
Well, in my case this issue was resolved by completely removing `prettier` dependencies from my project:

```js
"prettier": "^1.16.4",
"prettier-eslint": "^8.8.2",
"prettier-eslint-cli": "^4.7.1"
```

That's pretty strange behavior, though.

**Comment by sapegin** [comment]
Feel free to send a pull request with a fix 👾

**Comment by rgrwkmn** [comment]
@sapegin what commands do you run to generate the build for NPM?

**Comment by valleywood** [comment]
I had the same issue and I can confirm that removing dependencies to prettier solved the problem, would be grateful if someone could provide a fix for it though.

Here is the error that i get:
Error: import or require() statements can be added only by editing a Markdown example file

**Comment by rgrwkmn** [comment]
@valleywood are the prettier dependencies of your project or of react-styleguidist? [My bug reproduction repo does not have any other dependencies](https://github.com/rgrwkmn/styleguidist-require-import-bug/blob/master/package.json).

**Comment by valleywood** [comment]
> @valleywood are the prettier dependencies part of your project or of react-styleguidist? [My bug reproduction repo does not have any other dependencies](https://github.com/rgrwkmn/styleguidist-require-import-bug/blob/master/package.json).

 I had a reference to "prettier-eslint-cli": "4.7.1" in my project (not a dependency of react-styleguidist). I removed it, removed the node_modules folder and and re-run yarn and everything started working again.

Strange that it doesn't work for you despite not having this dependency 🤔

**Comment by rgrwkmn** [comment]
@valleywood can you clone my repro repo and try it? It should only take a minute to verify. Afaik nobody has confirmed that it also reproduces the bug for them:
```sh
git clone git@github.com:rgrwkmn/styleguidist-require-import-bug.git \
&& cd styleguidist-require-import-bug \
&& npm install \
&& npm run styleguidist
```

**Comment by valleywood** [comment]
@rgrwkmn Tried but got permission denied when cloning

**Comment by rgrwkmn** [comment]
@valleywood thanks for trying! I updated the commands to use HTTPS instead of SSH, try that if you have a minute.

**Comment by valleywood** [comment]
@rgrwkmn Confirmed, I can repeat the error with your solution. I also see that your node_modules folder doesn't contain prettier-eslint-cli (a lot of other stuff though :))

**Comment by timsnadden** [comment]
What fixed this for me was adding the following to `devServer` in the styleguidist config.

```
headers: {
	'Access-Control-Allow-Origin': '*',
},
```

**Comment by rgrwkmn** [comment]
@timsnadden thanks for the report! That must be having some interaction with other configs you're using because I added it to the styleguidist webpackConfig in my repro repo and it didn't fix the issue there. I also upgraded to 9.0.3 and the outcome is the same.

There is a difference in the package served by NPM and the code from running `npm compile` in the cloned github repo. If @sapegin can give me some insight into the build process for NPM I'll take a look. I didn't see any info on that in the [contributing docs](https://react-styleguidist.js.org/docs/development.html). 

For now my workaround is linking to my locally built version of styleguidist.
